PythonVisualizations/OrderedArray.py

```diff
@@ -219,7 +250,7 @@ def display(self):
             # create display objects for the associated Drawables
             n.display_shape, n.display_val = self.createCellValue(
                 i, n.val, n.color)
-            n.color = self.canvas.itemconfigure(n.display_shape, 'fill')
+            n.color = self.canvas.itemconfigure(n.display_shape, 'fill')[-1]
```

Comment:
Added the `[-1]` to get the fill color from the end of the tuple returned by itemconfigure.